Components/Avatar/Avatar.css

```diff
@@ -0,0 +1,95 @@
+.small-avatar{
+    height: 7rem;
+    width: 7rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+.medium-avatar{
+    height: 14rem;
+    width: 14rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+.large-avatar{
+    height: 20rem;
+    width: 20rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+
+.avatar-type{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--landing-hover-color);
+}
+
+iframe{
+    padding: 1rem 2rem;
+    width: 110rem;
+    height: 22rem;
+    overflow-y: scroll;
+}
+
+.carbon-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.avatar-profile{
+    height: 14rem;
+    width: 14rem;
+    border-radius: 10rem;
+}
+
+.avatar-demo{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 3rem 4rem;
```

Comment:
For consistency in design, you would want to add a global variable for padding as well, something like 
--space-small:1rem
--space-xlarge:3rem

Reply:
okay will that in consideration for further use.